spikeinterface/toolkit/postprocessing/principal_component.py

```diff
@@ -580,6 +605,9 @@ def compute_principal_components(waveform_extractor, load_if_exists=False,
         If True, waveforms are pre-whitened
     dtype: dtype
         Dtype of the pc scores (default float32)
+    sparsity: dict or None
```

Comment:
I think we should unify the behavior by using `sparsity_dict` (see `epxort_to_phy`)

Reply:
I don't k now what this is. I used the same sparsity argument as in the WaveformExtractor, or maybe I missed something. I'll have a look. On a side node, the computation of the PCA models could (should ?) be parallelized over jobs if possible

Reply:
Yes it's currently quite slow and it should be parallelized